spec/features/profiles_spec.rb

```diff
@@ -9,8 +9,9 @@
   it 'is check user info in show page' do
     @user = create(:user)
     visit profile_path(id: @user.id)
-    expect(page).to have_content "#{@user.name}"
-    expect(page).to have_content "#{@user.email}"
+    expect(page).to have_content 'Данные пользователя'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.